cwms-data-api/src/test/java/cwms/cda/data/dao/location/kind/ProjectStructureIT.java

```diff
@@ -146,4 +157,52 @@ public static void deleteLocation(DSLContext context, String officeId, String lo
 			LOGGER.atFinest().withCause(ex).log("No data found for " + officeId + "." + locId);
 		}
 	}
+
+	public static void deleteLocationGroup(DSLContext context, Outlet outlet) {
+		LocationGroupDao locationGroupDao = new LocationGroupDao(context);
+		try {
+			locationGroupDao.delete(outlet.getRatingCategoryId().getName(), outlet.getRatingGroupId().getName(), true, OFFICE_ID);
+		} catch (NotFoundException e) {
+			LOGGER.atFinest().withCause(e).log("No data found for category:" + outlet.getRatingCategoryId().getName()
+													   + ", group-id:" + outlet.getRatingGroupId().getName());
+		}
+	}
+
+	public static <T extends CwmsDTOBase> void containsDto(List<T> outlets, T expectedDto, BiPredicate<T, T> identifier, BiConsumer<T, T> dtoMatcher) {
```

Comment:
These should probably just be matchers to maintain the expect test assert naming scheme in the test cases.

They're specific enough to outlets that keeping them in an "outlets" package makes sense. But I would prefer to see assert(doesNot)Contain(s)Dto in the test case itself.

Reply:
Not sure I'm understanding this comment.  What do you mean by matchers, and the assert contains bit?  Are you saying I should rename this and its inverse to assertContainsDto and assertDoesNotContainDto?

Reply:
Yeah. There are some other examples of the matchers in the test fixtures package.

They should also include a String message, nothing wrong with the default you have there, but seems worth making a bit more generic now.

Reply:
Oh, do you mean the assertMatch methods in DTOMatch class?  I could probably add something there for this so it's more reusable.

Reply:
Yep, don't get too hung up on generic - if it gets complicated, but we do want to match that style.

Reply:
I moved the methods into DTOMatch, they were already generic enough, just needed a scope change and rename.